crud/borders.lua

```diff
@@ -23,7 +23,7 @@ local function get_border_on_storage(border_name, space_name, index_id, field_na
 
     local space = box.space[space_name]
```

Comment:
See https://github.com/tarantool/crud/pull/244#discussion_r773835277

I need to be able to recognize "space not found" error so this case could be **optimally** processed (instead of doing excessive "is space exist" check). I see two ways to recognize "space not found" error (at least while we're in scope of tarantool/errors): check error message or check error class. Now error system in crud is based on operations (`InsertError` etc). To make it less invasive and do not change current behavior, I decided to do it this way

Reply:
There was also a discussion in https://github.com/tarantool/crud/pull/244#discussion_r773389975